framework/src/modules/chain/submodules/process_transactions.js

```diff
@@ -18,6 +18,7 @@ const {
 	Status: TransactionStatus,
 	TransactionError,
 } = require('@liskhq/lisk-transactions');
+const BigNum = require('bignumber.js');
```

Comment:
why not use `@liskhq/bignum` library?

Reply:
Its not added as dependency for the framework in `2.0.0`. And don't want to add it just for this patch.

Reply:
We're using @liskhq/bignum in 2.2 @ManuGowda so we'll need to update when merging back :)